# I/O Topics
goal_reached_topic: "/local_planner/goal_reached" # Goal reached topic
trajectory_topic:   "/local_planner/trajectory"   # Local trajectory topic
local_pose_topic:   "/ground_truth/state"         # Local Pose Topic
command_topic:      "/cmd_vel"                    # Command topic

# Performance
update_rate_hz: 100.0 # Update rate in hz